llvm/lib/CodeGen/AsmPrinter/AsmPrinter.cpp

```diff
@@ -1095,23 +1095,23 @@ void AsmPrinter::emitBBAddrMapSection(const MachineFunction &MF) {
   OutStreamer->emitULEB128IntValue(MF.size());
   const Function &F = MF.getFunction();
 
-  // During code-gen, LLVM can merge many BasicBlocks into a single
-  // MachineBasicBlock. Unfortunately MachineBasicBlock::getBasicBlock() will
-  // return the only the first BasicBlock in a sequence of merged blocks, so we
-  // have to find the merged blocks via other means.
+  // LLVM's codegen can can merge multiple BasicBlocks into a single
+  // MachineBasicBlock. Unfortunately, MachineBasicBlock::getBasicBlock() only
+  // returns the first BasicBlock in the merged sequence, so we have to find
+  // the other corresponding BasicBlock(s) (if any) in the merged sequence
+  // another way. We do so in two steps:
   //
-  // To find all of the corresponding BasicBlocks for a MachineBasicBlock, MBB:
+  // 1. We create a set, MergedBBs, which is the set of BasicBlocks that are
```

Comment:
Super nerdy comment, I know, but I think we should semi-markdown these and indent the list. Thoughts?